sync-core/src/main/java/com/cloudant/sync/query/UnindexedMatcher.java

```diff
@@ -258,7 +265,7 @@ private boolean executeSelectorTree(QueryNode node, DocumentRevision rev) {
     private boolean valueCompare(Object actual, String operator, Object expected) {
         boolean passed;
 
-        if (operator.equals("$eq")) {
+        if (operator.equals("$eq") || operator.equals("$in")) {
```

Comment:
A comment here saying something like "$in implies $eq for each value in its array (caller breaks array into separate values)" could be useful, as it's a bit confusing given you'd expect `expected` to be an array if you were reading in isolation.